plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1309,6 +1310,21 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
     return true
   }
 
+  private fun isPointAboveHorizon(
+    pixel: ScreenCoordinate
+  ): Boolean {
+    if (mapProjectionDelegate.getMapProjection() != MapProjection.Mercator) {
+      return false
+    }
+    // Prevent drag start in area around horizon to avoid sharp map movements
+    val topMapMargin = 0.04 * mapTransformDelegate.getSize().height
```

Comment:
Wonder what are these constants? Some heuristic?

Reply:
@astojilj could you give more context here? It would be good to add some documentation to make the code more readable.

Reply:
@pengdev also let's make 100% sure that we're using regular pixels here everywhere and not mess up with logical pixels (dp).

Reply:
These coordinates are using the same pixel units, as we do all the conversion before passing to gl-native.

Reply:
Yes, the this here is in pixels. It is heuristic based on behavior on mobile device (video here https://github.com/mapbox/mapbox-maps-android/pull/927/files/e9eb47226c83b72ba015832558751b6297ff5741#r760989741) that should be safe to remove. Dead gesture area under horizon on large tablet screens is likely not desired.